docs/GPU-Performance.rst

```diff
@@ -163,6 +163,7 @@ We record the wall clock time after 500 iterations, as shown in the figure below
 .. image:: ./_static/images/gpu-performance-comparison.png
    :align: center
    :target: ./_static/images/gpu-performance-comparison.png
+   :alt: A G. P. U. performance chart which is a record of the wall clock time after 500 iterations for Higgs, epsilon, Bosch, Microsoft L. T. R, Expo and Yahoo L. T. R. and bin size of 63 performs comparitively better.
```

Comment:
```suggestion
   :alt: A performance chart which is a record of the wall clock time after 500 iterations on G P U for Higgs, epsilon, Bosch, Microsoft L T R, Expo and Yahoo L T R and bin size of 63 performs comparitively better.
```

Based on this article you shared (https://accessibility.psu.edu/abbreviations/, thanks for sharing that!), I think that for these `alt` tags all acronyms should just use spaces and not full stops.

> When writing ALT tags with acronyms, add spaces between characters (e.g. <alt="P S U" >).

I also think that for this particular one, the "GPU" part should be moved to the middle of the sentence so it's clear that "AGPU" is not a meaningful acronym.